contact.html

```diff
@@ -1,6 +1,7 @@
 ---
 title: Contact
 layout: default
+meta-description: Feel free to talk with and participate in the IRCv3 Working Group on irc.freenode.net/#ircv3
```

Comment:
Yeah, I was thinking about that but since it's just a meta description (and likely no clickable links), I think leaving it at this should be fine. Hopefully most of the people checking us out know Freenode (or just IRC in general) well enough anyways.